.github/workflows/build-and-test.yml

```
@@ -2,22 +2,20 @@ name: Build and Test
 
 env:
-  GO_VERSION: '1.21'
+  GO_VERSION: '1.25.0'
   NODE_VERSION: '18'
 
 jobs:
   test-go:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)